packages/react-components/src/components/PictureInPictureInPicture/PictureInPictureInPicture.tsx

```diff
@@ -48,16 +60,33 @@ export const _PictureInPictureInPicture = (props: _PictureInPictureInPictureProp
  */
 const PictureInPictureInPictureContainer = (props: {
   primaryView: ReactChild;
-  secondaryView: ReactChild;
+  secondaryView?: ReactChild;
   onClick?: () => void;
+  ariaLabel: string;
 }): JSX.Element => (
-  <div style={tileContainerStyles} onClick={props.onClick}>
+  <aside
```

Comment:
For my learning, why is `aside` used over `div` :)?
Nvm haha it's in the description